packages/graphql/src/resolvers/marketplace/listings.js

```diff
@@ -142,13 +142,19 @@ export default async function listings(
   }
 
   let ids = [],
-    totalCount = 0
+    totalCount = 0,
+    discoveryError = false
 
   if (contracts.discovery) {
-    const discoveryResult = await searchIds(search, filters)
-    ids = discoveryResult.ids
-    totalCount = ids.length
-  } else {
+    try {
+      const discoveryResult = await searchIds(search, filters)
+      ids = discoveryResult.ids
+      totalCount = ids.length
+    } catch (e) {
+      discoveryError = true
```

Comment:
Should we log an error in the console so that we know it failed ?

Reply:
Sure... just added that in